### PR TITLE
Fix `ProgressBar.update_speed`

### LINKED ISSUE #361: ProgressBar extension doesn't show correct `iters/sec` in ppe==0.5.0
The `ProgressBar` extension shows `iters/sec` as well as the progress bar, but I noticed that in ppe==0.5.0 it becomes always `inf iters/sec`.

### ppe==0.5.0

Minimum reproduction code:
```python
# progress.py (based on https://git.io/JzHP3)
import pytorch_pfn_extras as ppe
import time

print("PPE =", ppe.__version__)

max_epochs = 3
iters_per_epoch = 20
manager = ppe.training.ExtensionsManager({}, {}, max_epochs, iters_per_epoch=iters_per_epoch)
manager.extend(ppe.training.extensions.ProgressBar(training_length=None, update_interval=1, bar_length=40))

for epoch in range(max_epochs):
    for _ in range(iters_per_epoch):
        with manager.run_iteration():
            pass
        time.sleep(0.1)
```

Result:
```
$ python progress.py
PPE = 0.5.0
     total [####################....................] 51.67%
this epoch [######################..................] 55.00%
        31 iter, 1 epoch / 3 epochs
       inf iters/sec. Estimated time to finish: 0:00:00.
```

### ppe==0.4.5 (expected behavior)
It shows a correct iters/sec.
```
$ python progress.py
PPE = 0.4.5
     total [#################.......................] 43.33%
this epoch [############............................] 30.00%
        26 iter, 1 epoch / 3 epochs
    10.825 iters/sec. Estimated time to finish: 0:00:03.140780.
```

### PR DESCRIPTION
**Pull request by asi1024**
Fix #361.

### CODE REVIEW

**Review by kmaehashi — CHANGES_REQUESTED** [review]
LGTM except for minor comment.

**Review by kmaehashi — COMMENTED** [review]
~test comment~

File: tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_progress_bar.py [hunk 1]
```diff
@@ -21,8 +23,11 @@ def test_run():
...
                 if manager.iteration < 2:
                     continue
-                status = '{} iter, {} epoch / {} epochs'.format(
-                    manager.iteration, epoch, max_epochs)
-                assert status in out.getvalue()
+                status = out.getvalue()
+                assert '{} iter, {} epoch / {} epochs'.format(
+                    manager.iteration, epoch, max_epochs) in status
+                iters_per_sec = re.findall(r'[0-9]+\.[0-9]* iters/sec', status)[-1]
+                assert 8 <= float(iters_per_sec[:-10]) <= 12
```
Comment (kmaehashi) [inline tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_progress_bar.py:33]: Could you write this with `re.search` with grouping for better readablity?

Comment (asi1024) [reply tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_progress_bar.py]: Fixed the regexp pattern to avoid dirty trimming.

**Review by kmaehashi — APPROVED** [review]
LGTM!

**Pull request merged**